app/main.py

```diff
@@ -1,7 +1,23 @@
+import requests
+
+
 def get_weather() -> None:
```

Comment:
Issue: The function 'get_weather' is annotated to return None (-> None), but it actually returns a string in both success and error cases. You should update the return type annotation to '-> str' to accurately reflect the function's behavior.